pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -53,6 +54,7 @@ type Response struct {
 
 type Client struct {
 	*client.Client
+	*generated.TestgroupClient
```

Comment:
repeating this on latest diff so it doesn't get lost:

It would be nice if this test didn't have to change anything in the client directory.